tests/plugin_integration.suite.dist.yml

```diff
@@ -0,0 +1,36 @@
+actor: Plugin_integrationTester
```

Comment:
I would create a suite dedicated to WPML integration only, not to general plugin integration.
It would simplify its maintenance.

Reply:
Yeah, I was torn between that and having X suites for each plugin we may do this with in the future. Divi, Elementor, Yoast etc etc.... 
But I'll single this out to WPML, I don't disagree.